server/index.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost/profiledb', { useNewUrlParser: true })
+    .then(() => console.log('Connected to MongoDb...'))
+    .catch(err => console.error('Could not connect to MongoDB', err))
+
+const profileSchema = new mongoose.Schema({
+    name: String,
+    username: String,
+    email: String
+})
+
+const User = mongoose.model('Profiles', profileSchema);
```

Comment:
call this Users

Reply:
look up naming convention for collections in Mongodb (should it be plural or not)